### PR TITLE
Update package.json to include engines

### PR DESCRIPTION
**Pull request by gideongrinberg**
Updates package.json to force users to use npm version 15+. Fixes a common issue on the discord. Should be updated with better error messages seeing as a lot of not-so-techy-savvy people have been having problems.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "name": "nodepolus2",
   "version": "1.0.0",
   "description": "",
+  "engines" : {"node" : ">=15.0.0"},
```
Comment (roobscoob) [inline package.json]: This should be 15.0.1

**Comment by codyphobe** [comment]
This won't work as `engine-strict` is a directive for a user's global npm config.

You need to create an `.npmrc` file at the project root with these contents:

```
engine-strict=true
```

**Comment by gideongrinberg** [comment]
My bad, reading a old NPM doc version

**Comment by gideongrinberg** [comment]
Engine strict should be left in the package.json if only a warning is required.

**Comment by codyphobe** [comment]
`engine-strict` is not a property of `package.json` so it won't be read anyways. `engineStrict` *was* a property which did the job of the `.npmrc` fix mentioned above but that has been deprecated for some time.

**Comment by codyphobe** [comment]
Could you also format the `engines` object to match the format of the rest of the file

**Comment by gideongrinberg** [comment]
Sorry about all the commits. That was dumb 🤦.

**Pull request merged**